src/main/java/org/broadinstitute/hellbender/tools/walkers/concordance/Concordance.java

```diff
@@ -0,0 +1,244 @@
+            } else {
+                // the eval variant matches truth's position but their alleles don't match
+                // e.g. genotype or alleles don't match
+                if (variant.isSNP()) {
+                    snpFalsePositives++;
+                    snpFalseNegatives++;
+                } else {
+                    indelFalsePositives++;
+                    indelFalseNegatives++;
+                }
+                writeThisRecord(new VariantStatusRecord(variant, FALSE_POSITIVE, evalSampleName), variant);
+            }
+
+            if (truthIterator.hasNext()) {
+                currentTruthVariant = getNextNonSVTruthVariant();
+            } else {
+                exhaustedTruthVariants = true;
+            }
+
+            return;
+        }
+
+        // case 3: truth got ahead of eval; the eval variant must be a false positive if not filtered
+        if (variantContextComparator.compare(variant, currentTruthVariant.get()) < 0) {
+            if (variant.isNotFiltered()) {
+                if (variant.isSNP()) { snpFalsePositives++; } else { indelFalsePositives++; }
+                writeThisRecord(new VariantStatusRecord(variant, FALSE_POSITIVE, evalSampleName), variant);
+            }
+
+            // we don't care about true negatives - don't record them in the table
+            return;
+        }
+    }
+
+    // ugly code alert
+    // this method just moves the iterator forward but also ensures that it skips SV's
+    // assumes that the caller already checked the truth iterator has next
+    // TODO: extend the truth iterator such that the iterator.next() automatically skips SVs
+    private Optional<VariantContext> getNextNonSVTruthVariant(){
+        VariantContext nextVariant;
+        do {
+            // if the truth vcf ends with an SV, truthIterator.next() may return null
+            // so make sure that we didn't reach the end of truth
+            if (! truthIterator.hasNext()){
+                exhaustedTruthVariants = true;
+                return Optional.empty();
+            }
+            nextVariant = truthIterator.next();
+        } while (nextVariant.getStructuralVariantType() != null);
+
+        return Optional.of(nextVariant);
+    }
+
+    private void writeThisRecord(final VariantStatusRecord record, final VariantContext variant){
+        try {
+            variantStatusWriter.writeRecord(record);
+        } catch (Exception e) {
+            throw new UserException(String.format("Encountered an IO exception writing a record at chrom %d, pos ",
+                    variant.getContig(), variant.getStart()), e);
+        }
+    }
+
+    @Override
+    public Object onTraversalSuccess(){
+        if (! exhaustedTruthVariants){
+            // the remaining variants in truth are false negatives
+            if (currentTruthVariant.get().isSNP()) { snpFalseNegatives++; } else { indelFalseNegatives++; }
```

Comment:
Extract this repeated chunk of code as `private void process falseNegative(final VariantContext vc)`.